constraint_utils/constraint_basics.py

```diff
@@ -722,6 +746,16 @@ def build_plateau_models_pop(ht: hl.Table, weighted: bool = False) -> Dict[str,
 
 
 def get_all_pop_lengths(ht, prefix: str = 'observed_', pops: List[str] = POPS, skip_assertion: bool = False):
+    """
+    Get the minimum size of the arrays under observed variants column for each population.
```

Comment:
Maybe something like this? You may be able to think of something better, but it doesn't have to be the observed variants column, that is just the default.

```suggestion
    Get the minimum array length for each annotation in `ht` that is defined by the combination of `prefix` and each population in `pops`.
```